cli/pkg/cmd/upgrade/upgrade.go

```diff
@@ -0,0 +1,110 @@
+package upgrade
+
+import (
+	"cli/pkg/cmd/config"
+	"cli/pkg/console"
+	"cli/pkg/helm"
+	"cli/pkg/kube"
+	"cli/pkg/workspace"
+	"fmt"
+
+	"github.com/airyhq/airy/infrastructure/lib/go/k8s/util"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	approve    bool
+	version    string
+	Version    string
+	UpgradeCmd = &cobra.Command{
+		Use:   "upgrade [workspace directory]",
```

Comment:
Let's keep the workspace directory as a flag `--workspace` to keep it consistent with other commands that operate on an existing workspace.